lib/Form/Type/FieldType/TimeFieldType.php

```diff
@@ -44,21 +45,21 @@ public function getParent()
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->addModelTransformer(new FieldValueTransformer($this->fieldTypeService->getFieldType('eztime')));
+        $builder
+            ->addViewTransformer(new FormattedTimeToTimestampTransformer($options['with_seconds']))
```

Comment:
Are you sure `with_seconds` key will be always present?

Reply:
Yes. It has default value always set.